docs/static/governance/avm-standard-github-labels.csv

```diff
@@ -6,7 +6,7 @@ Needs: Author Feedback :ear:,Awaiting feedback from the issue/PR author,F18A07
 Needs: External Changes :hammer_and_pick:,When an issue/PR requires changes that are outside of the control of the module. e.g. to an RP.,DE389D
 Needs: More Evidence :balance_scale:,We are looking for more evidence to make a decision on this,F64872
 Needs: Triage :mag:,Maintainers need to triage still,FBCA04
-Needs: Module Owner :mega:,This module needs an owner to develop or maintain it,FF0019
+Needs: Module Owner :mega:,"This module needs an owner to develop or maintain it, or the module owner needs to review a PR",FF0019
```

Comment:
While this is factually true, I think this is very confusing. I see two options:
1. We change the sentence to this: In the AVM repository: this module needs an owner to develop or maintain it. In the BRM repository: the module owner needs to review a PR.
2. We leave it as it was, as it's primarily intended to be used in the AVM repo - AFAIK, we don't have any automation in place to periodically reapply these in the BRM repo.

Reply:
I like the first option and will go for it.